### ISSUE TITLE
Use media queries instead of max-scale

### ISSUE THREAD

**Issue by peteramckay** [roscoelabs/beams#8]
Skeleton contributors recently had a good discussion about use of max-scale, at https://github.com/dhg/Skeleton/issues/173

After reading up on it, I'm really thinking the method is really kinda antiquated from the now-early days of responsive design. (Hard to believe it's been a few years now!) Could probably achieve same benefits using media queries based on device orientation.

TL;DR benefits of max-scale: Prevents weird resizing of elements when mobile devices change orientation.

TL;DR cons of max-scale: Prevents zooming. Hence it may impair usability for visually impaired users.

**Comment by peteramckay** [comment]
We've now added meta tag in previous comment to index.html, mirroring recent change in Skeleton. A hat tip to Dave Gamache and the rest of that community for their spirited discussion of the issue.
